content/_data/changelogs/lts.yml

```
@@ -13790,6 +13790,50 @@
     message: |-
       Avoid queue items being lost if the node disconnects at a bad time during allocation.
 
+- version: '2.528.2'
+  date: 2025-11-12
+  changes:
+  - type: rfe
+    category: rfe
+    authors:
+      - lemeurherve
+    pr_title: "fix: take in account Jenkins versioning format in versionLT and Compare-VersionLessThan"
+    references:
+      - url: https://github.com/jenkinsci/docker/pull/2099
+        title: Docker pull request 2099
+    message: |-
+      Update <code>versionLT</code> and <code>Compare-VersionLessThan</code> to account for Jenkins versioning formating.
```

```suggestion
      Update <code>versionLT</code> and <code>Compare-VersionLessThan</code> to account for Jenkins versioning formating in `jenkins-support` scripts.
```
A bit more precise. (Did not find any reference to those scripts in the README)

---

@kmartens27 something went wrong, it's not in the current change, FYI.

---

I went to fix a typo and must have done it prior to the change taking effect. I've gone ahead and updated the message entirely to provide the further precision and typo correction.